.github/workflows/ci.yml

```diff
@@ -196,15 +196,30 @@ jobs:
             backend: cpp
             env: { MACOSX_DEPLOYMENT_TARGET: 10.14 }
 
+        exclude:
+          # fails github.com/cython/cython/issues/3450
+          - os: windows-2019
+            python-version: 3.9
+
+          # fails due to lack of a compatible compiler
+          - os: windows-2019
+            python-version: 2.7
+          - os: windows-2019
+            python-version: 3.4
+
+          # cpp specific test fails
+          - os: windows-2019
+            python-version: 3.5
+            backend: cpp
+
     # This defaults to 360 minutes (6h) which is way too long and if a test gets stuck, it can block other pipelines.
     # From testing, the runs tend to take ~20 minutes, so a limit of 30 minutes should be enough. This can always be
     # changed in the future if needed.
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
probably there's need to change the comment

Reply:
I made it 60 because of [this](https://github.com/0dminnimda/cython/runs/3218434800), but now I realized that it was not the tests that took so long, and my compensation from 45 to 60 was in vain, so 45 is enough.

Reply:
both resolved [here](https://github.com/cython/cython/pull/4324/commits/969f16f2500afbc55f2701031071b71a77d749c6)